patches/0001-Fix-and-improve-metadata-file-for-Flatpak.patch

```diff
@@ -0,0 +1,136 @@
++      install( FILES "${_SRCDIR}/tenacity.metainfo.xml"
+                DESTINATION "${_DATADIR}/metainfo" )
+    endif()
+ endif()
+diff --git a/help/tenacity.appdata.xml b/help/tenacity.appdata.xml
+deleted file mode 100644
+index 7464b721a..000000000
+--- a/help/tenacity.appdata.xml
++++ /dev/null
+@@ -1,43 +0,0 @@
+-<?xml version="1.0" encoding="UTF-8"?>
+-<component type="desktop">
+-  <id>org.tencacityaudio.Tenacity</id>
+-  <launchable type="desktop-id">tenacity.desktop</launchable>
+-  <project_license>GPL-2.0 and CC-BY-3.0</project_license>
+-  <metadata_license>CC0-1.0</metadata_license>
+-  <name>Tenacity</name>
+-  <summary>Audio editor and recorder. Nothing more, nothing less.</summary>
+-  <description>
+-    <p><b>Tenacity</b> is an easy-to-use multi-track audio editor and recorder for Windows, MacOS, GNU/Linux and other operating systems.</p>
+-	<p>It is based on the popular Audacity audio editor and is being developed by a wide, diverse group of volunteers. It ships with some additional features and respects your privacy.</p>
+-    <p>You can use Tenacity to:</p>
+-    <ul>
+-      <li><b>Record</b> from audio devices (real or virtual)</li>
+-      <li><b>Export / Import</b> a wide range of audio formats (extendible with FFmpeg)</li>
+-      <li><b>High quality</b> including up to 32-bit float audio support</li>
+-      <li><b>Plug-ins</b> providing support for VST, LV2, and AU plugins</li>
+-      <li><b>Scripting</b> in the built-in scripting language Nyquist, or in Python, Perl and other languages with named pipes.</li>
+-      <li><b>Editing</b> arbitrary sampling and multi-track timeline</li>
+-      <li><b>Accessibility</b> including editing via keyboard, screen reader support and narration support</li>
+-      <li><b>Tools</b> useful in the analysis of signals, including audio</li>
+-    </ul>
+-  </description>
+-  <categories>
+-    <category>AudioVideo</category>
+-    <category>Audio</category>
+-  </categories>
+-  <content_rating type="oars-1.1" />
+-  <url type="homepage">https://tenacityaudio.org/</url>
+-  <url type="bugtracker">https://github.com/tenacityteam/tenacity/issues</url>
+-  <url type="faq">https://manual.audacityteam.org/man/faq.html</url>
+-  <url type="help">https://manual.audacityteam.org/</url>
+-  <url type="translate">https://hosted.weblate.org/projects/tenacity/tenacity</url>
+-  <screenshots>
+-    <screenshot type="default">
+-      <image type="source">https://www.audacityteam.org/wp-content/uploads/2016/01/audacity-212-linux.png</image>
+-    </screenshot>
+-    <screenshot>
+-      <image type="source">https://www.audacityteam.org/wp-content/uploads/2015/11/Audacity-2-2-0-on-Linux.png</image>
+-    </screenshot>
+-  </screenshots>
+-  <update_contact>~tenacity/tenacity-discuss@lists.sr.ht</update_contact>
+-</component>
+diff --git a/help/tenacity.metainfo.xml b/help/tenacity.metainfo.xml
+new file mode 100644
+index 000000000..e8386d8f5
+--- /dev/null
++++ b/help/tenacity.metainfo.xml
```

Comment:
you do you need to do that? `rename-appdata` should just work

Reply:
It's just temporary. I just want to check whether it builds or not because it keeps erroring on my PC so I can't test locally sadly.